.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: "*/59 23 * * *"
```

Comment:
btw I can add cron config to the publish flow and it will release new version every 24 hours. wdyt?

Reply:
As a rule of thumb, I personally dont like GAing when no one's looking. it creates issues in prod that only users will pick up.

Why would we want that cron? to always pick up latest deps?

Reply:
there is a task for automating the release of corvid-types on every doc change. using a cron job will achieve the goal of automation releases.